systems/subscriber/subscriber-registry/pkg/db/model.go

```diff
@@ -0,0 +1,25 @@
+package db
+
+import (
+	"time"
+
+	"github.com/gofrs/uuid"
+)
+
+type Subscriber struct {
+	SubscriberID          uuid.UUID `gorm:"type:uuid;index"`
```

Comment:
Does gorm `index` tag enforce field uniqueness? Not so sure of that. Please, confirm if so. If not, please explicitly enforce the `primary key` constraint for subscirberID field.

Reply:
You're right.I have changed it to be unique and primaryKey .index only help in filtering data.